### PR TITLE
Add basic pagination for slow feeds

### LINKED ISSUE #136: Add pagination to Slow
Right now having `slow` feeds with 100+ posts freeze browser when you open them.

So we need to add pagination. Requirements:
1. As with pagination in `fast` we need to keep the current page in URL.
2. Adding new posts should not change posts in pages. So it is better to use `since` rather than `page` in URL.
3. In contrast with pagination in `fast` user need to see how many pages are in the list.
4. Users need not only `next page`, but also the last page link, etc.
5. The small `1, 2, 3` links are not very useful. We need something better in terms of UX.

### PR DESCRIPTION
**Pull request by pekac**
Fixes #136 

Added pagination functionality for slow feeds.

Changes made in:
- core/
    - router
    - slow
- web/
    - pages
    - stores

## Checklist

- [x] Don’t rush. Check all changes in PR again.
- [x] Run `pnpm test`.
- [x] Think about changing documentation.
  - If you added a script to `scripts/`, add a comment with a description.
  - If you added a new folder, add its description to the project’s `README.md`.
  - If you added config, describe how we use this tool in the config’s comment.
  - If you added something to the project’s architecture, describe it in the project’s `README.md`.
  - Try to focus on “why?”, not “how?”.
- [x] If you added a new dependency, check our [requirements](../README.md#dependencies).
- [x] Think about testing
  - If you added a feature, add unit tests.
  - If you added a new state to the UI, add visual tests.
  - If you fixed the bug, think about preventing bug regression in the future.
- If you changed web client:
  - [x] Think about moving code to `core/`. What code will also be useful on other platforms?
  - [x] Run `pnpm size` and check the difference in the JS bundle size. Is it relevant to the changes? Change the limit in `web/.size-limit.json` if necessary.
  - [x] Think about keyboard UX. Is it easy to use the new feature with only one hand on a keyboard? Is it easy to understand what keys to press?
  - [x] Think about HTML semantics.
  - [x] Think about accessibility. Check a11y recommendations. Think about how screen reader users will use the tool. Is it easy to use on a screen with bad contrast?
  - [x] Think about translations. Will
  - [x] Think about right-to-left languages. What parts of the screen should be mirrored for Arabic or Hebrew languages?
- If you changed the colors token in the web client:
  - [x] Think about app loading styles inlined in `index.html`.
- If you changed `core/`:
  - [x] Think about making types more precise. Can you better explain data relations by type?
  - [x] Think about conflict resolution. We don’t need some very smart changing merging; just 2 changes of the same item on different clients should not break the database. What if the user changes an item on one machine and removes it on another?
  - [x] Think about log and storage migration.
- If you changed English translations:
  - [x] Change translation ID if you change the meaning of the text.

### CODE REVIEW

File: core/slow.ts [hunk 1]
```diff
@@ -120,67 +120,141 @@ let $posts = atom<SlowPostsValue>({ isLoading: true })
...
+
+export type SlowFeedsValue =
+  | { feeds: FeedValue[]; isLoading: false }
+  | { isLoading: true }
+
+let $slowFeeds = atom<SlowFeedsValue>({ isLoading: true })
+
+export const slowFeeds = readonlyExport($slowFeeds)
+
+let POSTS_PER_PAGE = 2
```
Comment (ai) [inline core/slow.ts:139]: It is very small number :)

**Comment by ai** [comment]
Looks good. Why it is have `Draft` mark?

**Comment by pekac** [comment]
Essentially, because of tests.

Need to check why 1 fails since that part of the logic wasn't touched.
And to ask about UI tests - since this PR adds new state to the UI should I cover that with tests?

**Comment by ai** [comment]
> since this PR adds new state to the UI should I cover that with tests?

Let’§ add visual tests when we will have a good design :)

**Pull request merged**